pkg/kv/store.go

```diff
@@ -33,6 +33,12 @@ var (
 	ErrTableNotActive      = errors.New("table not active")
 )
 
+// Precond Type for special conditionals provided as predicates for the SetIf method
+type Precond string
+
+// PrecondConditionalExists Conditional for SetIf which performs Set only if key already exists in store
+var PrecondConditionalExists = Precond("dd136a329c1f297d37d039d9ddd99699")
```

Comment:
Will be nice when we debug to see the value render with the meaning
```suggestion
var PrecondConditionalExists = Precond("ConditionalExists")
```